src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -48,14 +48,15 @@ See the LICENSE file in the project root for more information.
       <NativeLibrary Condition="$(NativeCodeGen) == ''" Include="$(IlcPath)/sdk/libRuntime.a" />
       <NativeLibrary Condition="$(NativeCodeGen) == 'cpp'" Include="$(IlcPath)/sdk/libbootstrappercpp.a" />
       <NativeLibrary Condition="$(NativeCodeGen) == 'cpp'" Include="$(IlcPath)/sdk/libPortableRuntime.a" />
-      <NativeLibrary Condition="$(NativeCodeGen) == 'wasm'" Include="$(IlcPath)/sdk/libbootstrappercpp.a" />
-      <NativeLibrary Condition="$(NativeCodeGen) == 'wasm'" Include="$(IlcPath)/sdk/libPortableRuntime.a" />
+      <NativeLibrary Condition="$(NativeCodeGen) == 'wasm'" Include="$(IlcPath)/sdk/libbootstrappercpp.bc" />
+      <NativeLibrary Condition="$(NativeCodeGen) == 'wasm'" Include="$(IlcPath)/sdk/libPortableRuntime.bc" />
     </ItemGroup>
 
     <ItemGroup>
-      <NativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.a" />
+      <NativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.a" Condition="$(NativeCodeGen) != 'wasm'" />
+      <NativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.bc" Condition="$(NativeCodeGen) == 'wasm'" />
       <NativeLibrary Include="$(IlcPath)/framework/System.Native.a" />
-      <NativeLibrary Include="$(IlcPath)/framework/System.Globalization.Native.a" />
+      <NativeLibrary Include="$(IlcPath)/framework/System.Globalization.Native.a" Condition="$(NativeCodeGen) != 'wasm'" />
       <NativeLibrary Include="$(IlcPath)/framework/System.IO.Compression.Native.a" />
```

Comment:
Do all these need to have .bc suffix?

Reply:
Probably (although until https://github.com/dotnet/corefx/issues/26107 happens, these files don't exist).

Reply:
Should be fixed now.